paddlenlp/transformers/tokenizer_utils_base.py

```diff
@@ -1690,6 +1689,10 @@ def convert_added_tokens(obj):
                 "Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained."
             )
 
+        # save all of related things into default root dir
+        if pretrained_model_name_or_path in cls.pretrained_init_configuration:
+            tokenizer.save_pretrained(default_root)
```

Comment:
Should this be triggered only when using `init_configuration` and not exist, just like that in `model.from_pretrained`